src/core_plugins/elasticsearch/lib/cluster.js

```diff
@@ -0,0 +1,137 @@
+import elasticsearch from 'elasticsearch';
+import { get, set, isEmpty, cloneDeep, pick } from 'lodash';
+import toPath from 'lodash/internal/toPath';
+import Boom from 'boom';
+
+import filterHeaders from './filter_headers';
+import { parseConfig } from './parse_config';
+
+export default class Cluster {
+  constructor(config) {
+    this._config = Object.assign({}, config);
+    this.errors = elasticsearch.errors;
+
+    createClients.call(this);
```

Comment:
Yes, this was left over from the addClientPlugins functionality and can now be removed. Just pushed a fix.